### PR TITLE
[PSDK] Define maximum SSID length properly

### PR DESCRIPTION
**Pull request by SigmaTel71**
## Purpose

Avoid using magic numbers for structures and ease the day for those who do not remember IEEE 802.11 specifications by heart.

JIRA issue: [CORE-6905](https://jira.reactos.org/browse/CORE-6905)
References: [Microsoft Learn](https://learn.microsoft.com/en-us/windows/win32/nativewifi/dot11-ssid)

<details><summary>IEEE 802.11</summary>

image
</details>

## Proposed changes

- Add a `#define` that matches by name with one used since day one of Visual Studio 2008 (VS2005 does not have Native Wi-Fi SDK)
- Change `DOT11_SSID.ucSSID` defined size from a magic constant to the new define

### CODE REVIEW

**Review by whindsaks — APPROVED** [review]

**Review by JoachimHenze — APPROVED** [review]
Approved.

**Review by HBelusca — APPROVED** [review]
It's interesting to see that in MS PSDK, some of these DOT11 definitions are not in windot11.h (this is the case of `DOT11_SSID`), but in wlantypes.h instead :thinking:

File: sdk/include/psdk/wlantypes.h [hunk 1]
```diff
@@ -7,6 +7,6 @@ typedef enum _DOT11_BSS_TYPE {
     dot11_BSS_type_any
 } DOT11_BSS_TYPE;
 
-
+#define DOT11_SSID_MAX_LENGTH 32
```
[the remark this answers is not in the document]
Comment (JoachimHenze) [reply sdk/include/psdk/wlantypes.h:10]: Never heard or seen that before.

[the remark this answers is not in the document]
Comment (SigmaTel71) [reply sdk/include/psdk/wlantypes.h:10]: That's not mentioned in our wiki's [Coding Style](https://reactos.org/wiki/Coding_Style). Even more, there's no section for preprocessor directives.

[the remark this answers is not in the document]
Comment (HBelusca) [reply sdk/include/psdk/wlantypes.h:10]: It's however pretty clear from any other header of ours, that it doesn't require any sort of entry in the coding style.

[the remark this answers is not in the document]
Comment (JoachimHenze) [reply sdk/include/psdk/wlantypes.h:10]: I do agree to Sigmatel in this regard. Especially due to the fact that the style that Hermes recommended here tends to become extremely confusing to read when multiple ifdefs are nested into each other.

[the remark this answers is not in the document]
Comment (binarymaster) [reply sdk/include/psdk/wlantypes.h:10]: Applied Hermes' suggestion because it makes the final diff smaller (+2 -2 vs. +2 -1) and cleaner.

**Pull request merged**